pgbadger

```diff
@@ -1586,7 +1597,7 @@ foreach my $logfile ( @given_log_files )
 
 	# Do not use split method with remote and compressed files, stdin, custom or journalctl command
 	if ( ($parallel_process > 1) && ($queue_size > 1) &&
-			($logfile !~ /\.(gz|bz2|zip|xz|bin)$/i) && ($logfile ne '-') &&
+			($logfile !~ $compress_extensions) && ($logfile ne '-') &&
```

Comment:
I am not sure if `bin` should be part of the same regex?

Reply:
Thanks Adrien, last commit f7dbec5 move .bin test in a separate condition clause.